data-imports/clean_utils.py

```diff
@@ -0,0 +1,40 @@
+def clean_addresses(table, column):
+    """ Returns a SQL statement cleaning an address column.
+        |table| and |column| are both strings.
+    """
+    # One line missing:
+    #    UPDATE {table} SET {column} = array_to_string(regexp_matches('(.*)(\d+)(?:TH|RD|ND|ST)( .+)'), '', {column} ) WHERE {column} ~ '.*(\d+)(?:TH|RD|ND|ST)( .+).*';
+    # No idea where regexp_matches is defined? Looks like it's a Postgresql command?
```

Comment:
Yup, `regexp_matches` is a Postgres function, [documented here](https://www.postgresql.org/docs/9.1/static/functions-string.html)